test/unit/nfts/ERC1155AutoGraphMinter.t.sol

```diff
@@ -129,6 +132,43 @@ contract UnitTestERC1155AutoGraphMinterTest is BaseTest {
         );
     }
 
+    function testMintForFreeWithExpiredJob() public {
+        Helper.TxParts memory parts = Helper.setupTx(vm, _privateKey, address(nft));
+
+        // mint
+        _autoGraphMinter.mintForFree(
+            parts.recipient,
+            parts.jobId,
+            parts.tokenId,
+            parts.units,
+            parts.hash,
+            parts.salt,
+            parts.signature,
+            address(nft),
+            block.timestamp
```

Comment:
shouldnt this be `parts.expiryToken` ?

Reply:
Good catch - yes, should be `parts.expiryToken`.